### ISSUE TITLE
Support for Dark Reader?

### ISSUE THREAD

**Issue by oskarsh** [evan-buss/Yin-Yang#2]
Hey another nice feature would be an integration with Dark Reader Extension. This is one of the most used Extension when it comes to Webpage Darkmode and is also open source.
You can send calls to the extension if you expose a API and can trigger Dark Mode. This involves alot of work and knowledge but would achieve a truly universal Dark Mode.

[DarkReader Repository](https://github.com/darkreader/darkreader)

**Comment by evan-buss** [comment]
Unless the extension exposes some API that allows you to toggle it, I don't really want to create some sort of patch-job that allows it to work. I feel that it would be more work than it's worth. 

There are also a bunch of different extensions that switch the browser CSS and it would be out-of-place to support a single browser extension. In my opinion, Yin-Yang should provide more general light-and-dark functionality for the operating system. Thank you for the suggestion though.

**Comment by oskarsh** [comment]
Yes I can totally understand and respect that decision. The Feature would be really nice but would cost to much work. You could leave this Issue open, I am sure someone who is willing to contribute to this Project will tackle this feature.
